<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Auth Key:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="authKeyLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Source-Target:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QKeySequenceEdit" name="sourceToTargetKeySequenceEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Target-Source:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QKeySequenceEdit" name="targetToSourceKeySequenceEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Source Language:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="sourceLanguageComboBox">
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chinese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Japanese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Russian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>German</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Target Language:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="targetLanguageComboBox">
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chinese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Japanese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Russian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>German</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
